src/components/Header/Header.styled.jsx

```diff
@@ -0,0 +1,74 @@
+import styled from "styled-components"
+
+export const Containerheder = styled.div` 
+position: fixed;
+height: 64px;
+top: 0;
+left: 0;
+z-index: 3;
+display: flex;
+padding: 0 12px;
+justify-content: space-between;
+align-items: center;
+background-color: white;
+box-shadow: 0px 2px 4px #00000026;
+flex-direction: row;
+width: 100%;
+`;
+
+export const Logoheader = styled.div` 
+width: 103px;
+min-width: 100px;
+height: 20px;
+margin-left: 16px;
+font-family: 'Montserrat-700';
+font-style: normal;
+font-weight: 700;
+font-size: 16px;
+line-height: 20px;
+letter-spacing: 0.08em;
+color: #000000;
+display: flex;
+flex-direction: row;
+padding: 0px;
+align-content: center;
+align-items: center;
+justify-content: space-between;
+`;
+
+export const NavheaderBlock = styled.div` 
```

Comment:
Try using more semantic tags instead of div

Reply:
I will try to do this.